Node/Managing Node-NPM versions.md

```diff
@@ -0,0 +1,126 @@
+## Node version management
+
+### Node release schedule
+
+Node.js has defined and predictable release plan. Depending on the Node release version number, the Node release can go through 3 different phases: current, active(LTS) and maintenance. Be aware that odd-numbered versions will not go through active and maintenance phases, while even numbered will go through them all.
+
+#### Current phase
+
+Releases within this phase incorporate most non-major changes that end up on `nodejs/node` main branch. New major releases are branched out every six months. Even-numbered versions are scheduled for release in April, while odd ones are scheduled for October.
+
+#### Active Long Term Support phase
+
+Releases within this phase include new features, bug fixes and updates that have been audited and determined to be stable for the release. With each odd-numbered release, previous even-numbered release will transition to LTS phase. Versions in this phase will receive active support for 12 months after they entered LTS phase.
+
+**You should always aim to use latest LTS version available on your project**
+
+#### Maintenance phase
+
+Releases within this phase will receive critical bug fixes and security updates for 18 months in case of LTS(even-numbered) and 2 months in case of regular(odd-numbered) releases after entering this phase.
+
+### Ensuring same version of Node is being used on all environments
```

Comment:
I would expend this chapter a bit. If we are using SSR (either Next.js or Angular Universal), then you also have to make sure that you use the same node version on both build server during the build step AND on production server when running the app.